cli/src/update-post-reboot.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
@SomwanshiAnjali
The general instructions for us are to have lesser batch scripts and more Python code.
Is it possible to convert this to Python code?
Also, move this code under dedicated upgrade directory:
`cli/src/upgrade/update-post-reboot.sh`

Reply:
@ypise It's good to move that file to a separate dir as you suggested but I think we can keep bash for now. Actually it was me who asked @SomwanshiAnjali to do just a PoC without any restrictions regarding implementation. I believe python will definitely come into play soon in scope of the next upgrade PR from Anjali.

Reply:
Addressed